- request:
    body: '{"plan_slug": "unittest", "expiration": null, "name": "Test Account"}'
    headers: !!python/object:requests.structures.CaseInsensitiveDict
      _store:
        accept: !!python/tuple [accept, application/json]
        accept-encoding: !!python/tuple [Accept-Encoding, 'gzip, deflate, compress']
        authorization: !!python/tuple [Authorization, !!python/unicode 'Basic cXhSU05jSWRlQToxZjBBVkNaUEpiUm5kRjlGTlNHTU9XTWZIOUtNVURhWA==']
        cache-control: !!python/tuple [cache-control, no-cache]
        content-length: !!python/tuple [Content-Length, !!python/unicode '69']
        content-type: !!python/tuple [content-type, application/json]
        user-agent: !!python/tuple [user-agent, myfc_id client]
    method: POST
    url: /organizations/api/identities/c3769912-baa9-4a0c-9856-395a706c7d57/accounts/
  response:
    body: {string: '"ServiceAccount for service identity_client and account ''Test
        Account'' already exists and is active. Conflict"'}
    headers: {allow: 'GET, POST, HEAD, OPTIONS', connection: keep-alive, content-length: '110',
      content-type: application/json, date: 'Tue, 25 Jun 2013 23:28:05 GMT', p3p: CP="NON
        DSP COR CURa TIA", server: nginx/0.7.67, vary: 'Authenticate, Accept'}
    status: {code: 409, message: CONFLICT}
